.github/release-drafter.yml

```diff
@@ -0,0 +1,48 @@
+# Configuration for Release Drafter: https://github.com/toolmantim/release-drafter
+name-template: $NEXT_PATCH_VERSION
+tag-template: $NEXT_PATCH_VERSION
+version-template: $MAJOR.$MINOR-beta-$PATCH
+
+# Emoji reference: https://gitmoji.carloscuesta.me/
+categories:
+  - title: 🚨 Removed
+    label: removed
+  - title: ⚠️ Deprecated
+    label: deprecated
+  - title: 🚀 New features and improvements
+    label: enhancement
+  - title: 🐛 Bug Fixes
+    label: bugfix
```

Comment:
the default label is `bug` we went with `fix` but `bugfix` seems fine :sweat_smile:

Reply:
`bug` always raises a question whether it is "added a new bug" or "fixed a bug" :)